docs/references/general/rawacf.md

```diff
@@ -1,3 +1,111 @@
+| *ercod*                 | *None*   | short     | Error Code                                                                              |
+| *stat.agc*              | *None*   | short     | AGC status word                                                                         |
+| *stat.lopwr*            | *None*   | short     | LOPWR status word                                                                       |
+| *noise.search*          | *None*   | float     | Calculated noise from clear frequency search                                            |
+| *noise.mean*            | *None*   | float     | Average noise from clear frequency band                                                 |
+| *channel*               | *None*   | short     | Channel number, used to denote different Tx/Rx channels on Stereo radars, and to denote changes in radar operating parameters between scans, e.g. alternating between 2 frequencies scan-to-scan      |
+| *bmnum*                 | *None*   | short     | Beam number, zero based indexing                                                        |
+| *bmazm*                 | *None*   | float     | Beam azimuth                                                                            |
+| *scan*                  | *None*   | short     | Scan flag, switches between 0 to +/-1. For some radars, the sign indicates scan direction. |
+| *offset*                | *us*     | short     | Offset between channels for a stereo radar                                              |
+| *rxrise*                | *us*     | short     | Receiver rise time                                                                      |
+| *intt.sc*               | *s*      | short     | Number of seconds of integration time                                                   |
+| *intt.us*               | *us*     | int       | Number of microseconds of integration time                                              |
+| *txpl*                  | *us*     | short     | Transmitted pulse length                                                                |
+| *mpinc*                 | *us*     | short     | Multi-pulse increment                                                                   |
+| *mppul*                 | *None*   | short     | Number of pulses in the sequence                                                        |
+| *mplgs*                 | *None*   | short     | Number of lags estimated from the sequence                                              |
+| *mplgexs*               | *None*   | short     | Number of lags used for tauscan. This field originates from `fittex` algorithm and is not used in RAWACF file format. | 
+| *ifmode*                | *None*   | short     | A flag indicating that the receiver voltage samples were obtained by digitizing the signal at an intermediate frequency (and then digitally mixed to baseband) instead of mixing to baseband and then digitizing (`ifmode=0`).         |
```

Comment:
```suggestion
| *ifmode*                | *None*   | short     | A flag indicating that the receiver voltage samples were obtained by digitizing the signal at an intermediate frequency (and then digitally mixed to baseband) instead of directly sampling the RF signals (`ifmode=0`).         |
```

Might need confirmation from some others, but this is better as the "RF mode" didn't involve any mixing.  I haven't read through this completely, but this quickly caught my eye.

Reply:
@egthomas @aburrell @asreimer @kevinkrieger

Reply:
Still waiting on some input here.  Likely a good input on this could be from @kevinkrieger. Thoughts?

Reply:
I'm sorry but I can't say I'm familiar with the 'ifmode' parameter

Reply:
OK, sounds good, just wondering if it had been implemented in Borealis or not.  If this is only an MSI-style parameter (fhe, fhw, cve, cvw, ade, adw) then my suggestion is a better description.

Reply:
I'm looking at a file from Saskatoon from January 2019 that has `ifmode=-1`

Reply:
It seems like nobody is here sure about the definition of `ifmode`. I'll send an email to a few PIs to see what I can find out.

Reply:
I tried contacting some PIs about this but got no responses. I will commit @ksterne's suggestion and then we can correct it in the future if we get any further input.